tool/parse_cli.c

```diff
@@ -202,50 +146,41 @@ static
 void parse_genservercert_cli(int argc, char **argv, struct cli_params *params)
 {
     params->rootcert = "root_cert.bin";
-    params->rootpriv = "root_priv.bin";
+    params->keypair = "root_keypair.bin";
     params->servercert = "server_cert.bin";
-    params->serverpriv = "server_priv.bin";
-    params->serverpub = "server_pub.bin";
+    params->serverkeypair = "server_keypair.bin";
     params->certreserved = NULL;
     const char *usage_str = "Generate server's certificate.\n\n"
         "Usage: %s %s [-h] [-r <file>] [-p <file>] [-v <file>] [-b <file>] [-x <file>] [-c <file>]\n"
         "\tOptions:\n"
         "\t\t-h --help                    Display this message.\n"
         "\t\t-r --rcert                   Root certificate input location [default = root_cert.bin]\n"
-        "\t\t-p --rpriv                   Root private key input location [default = root_priv.bin]\n"
-        "\t\t-v --serverpriv              Server private key input location [default = server_priv.bin]\n"
-        "\t\t-b --serverpub               Server public key input location [default = server_pub.bin]\n"
+        "\t\t-k --rkeypair                Root key pair input location [default = root_keypair.bin]\n"
+        "\t\t-s --skeypair                Server key pair input location [default = server_keypair.bin]\n"
```

Comment:
Can we make this default `server_keys.asn1.bin`? And, again, we'll have to propagate this to all the other places that use this.